vendors.md

```diff
@@ -25,28 +25,28 @@ This means vendors can offer different types of services around OpenZaak. The mo
+As a vendor you also benefit from knowledge and the continued development and support within the OpenZaak community. Collabroation on the OpenZaak codebase is done in an open source project by an open and active community of municipalities, vendors and software developers and welcoming everyone who wants to participate and contribute towards making OpenZaak better. On the [developer](/developers) section and our [GitHub repository](https://github.com/open-zaak/open-zaak) you can read [how to get started](https://open-zaak.readthedocs.io/en/latest/installation/index.html#installation-index) and how to [contribute](https://github.com/open-zaak/open-zaak/blob/master/CONTRIBUTING.md).
 
 ## Where do I start?
 
-The best way to start is to  sign up sign up to the [OpenZaak mailinglist](https://lists.publiccode.net/mailman/postorius/lists/openzaak-discuss.lists.publiccode.net/) to get al the latest news around the OpenZaak codebase and community. Furthermore you can join the [OpenZaak community calls](#) to learn more and meet the OpenZaak community.
+The best way to start is to sign up to the [OpenZaak mailing list](https://lists.publiccode.net/mailman/postorius/lists/openzaak-discuss.lists.publiccode.net/) to connect with the OpenZaakcommunity. Furthermore you can join the [OpenZaak community calls](#) to learn more and meet the OpenZaak codebase and community.
 
-We also invite you to [join and participate our community](/community) and learn how other municipalities and vendors are using OpenZaak. 
+We also invite you to [join and participate our community](/community) to learn how other municipalities and vendors are using OpenZaak. 
 
 ## How can I integrate OpenZaak into my product or service?
 
-The answer to this question largely depends on what kind of vendor you are and where you can offer the most value. Whether you are a full service IT vendor or software vendor. There are vendors who have decided to implement the ZGW API's from sratch and not relying on OpenZaak and there are vendors who integrated the OpenZaak codebased into their own product. Some vendors focus on hosting the back-end towards municipalities or even other vendors.
+The answer to this question largely depends on what kind of vendor you are and what value you want to offer, i.e. whether you are a full service IT vendor or software vendor. There are vendors who have decided to implement the ZGW API's from scratch and to not rely on OpenZaak and there are vendors who have integrated the OpenZaak codebased into their own product. Some vendors focus on hosting the back-end towards municipalities or even other vendors.
```

Comment:
Can we drop the _id est_ in this? There may be more types of vendors, the _i.e._ seems restrictive here.

Reply:
Oh, I was actually going for e.g. but it slipped my mind. Would that be better or should we just skip it completely?

Reply:
yeah, "e.g." or "for instance" would work, but I think just skip it.